src/main/webapp/scripts/tests/analyze-test.js

```diff
@@ -0,0 +1,67 @@
+const VIDEO_LINK = "https://www.youtube.com/watch?v=testvideoid";
+const SCORE = 0.05;
+
+QUnit.module('Updating DOM after analyzing video');
+QUnit.testStart(() => {
```

Comment:
nit: Since you are stubbing fetch in every test and restore it afterwards, you can also extract that behavior into beforeEach and afterEach just like you did here with testStart.